code/game/gamemodes/technomancer/spells/audible_deception.dm

```diff
@@ -34,18 +34,18 @@
 		"Gun Firing"			=	'sound/weapons/gunshot.ogg',
 		"Desert Eagle Firing"	=	'sound/weapons/deagle.ogg',
 		"Rifle Firing"			=	'sound/weapons/rifleshot.ogg',
-		"Rifle Firing 2"		=	'sound/weapons/svd_shot.ogg',
-		"Sniper Firing"			=	'sound/weapons/sniper.ogg',
+		"Sniper Rifle Firing"	=	'sound/weapons/svd_shot.ogg',
```

Comment:
This seems inconsistently spaced with most of the rest, after you touched it.

Reply:
That's just Github making it look stupid.